src/styles/components/_sandpack.scss

```diff
@@ -0,0 +1,163 @@
+// the article container
+.sandpack {
+  header {
+    color: var(--foreground-color);
+
+    h2 {
+      position: relative;
+      font-size: 1.5625rem; // 25px/16
+      font-family: Viga, sans-serif;
+      line-height: 2.125rem; // 34px/16
+      margin-inline: auto;
+
+      &::after {
+        content: "";
+        position: absolute;
+        width: 2rem;
+        height: 2.5rem;
+        background-image: url("/images/cmt_inventory_arrow.svg");
+        transform: translateX(0.5rem) rotate(-30deg);
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover;
+
+        @media (prefers-color-scheme: dark) {
+          filter: invert(1);
+        }
+      }
+
+      @media (min-width: $breakpoint-increase-content-font-size) {
+        font-size: map.get($font-sizes, "md");
+        line-height: 2.5rem; // 40px/16
+
+        &::after {
+          width: 2.5rem;
+          height: 3rem;
+        }
+      }
+    }
+  }
+}
+
+// because we have no control over the auto-generated sandpack class names,
+// I turned off the kebab-case linting error for certain selectors
+
+// sandpack component wrapper
+/* stylelint-disable-next-line selector-class-pattern */
+// .sp-wrapper.light.sp-c-fVPbOs-gMQIch-variant-light {
+//   height: 30rem;
+// }
+
+// component editor wrapper
+/* stylelint-disable-next-line selector-class-pattern */
+// .sp-layout.sp-c-ctabmA {
+//   height: 30rem;
+// the height of the editor can be passed into the options object on the Sandpack component!
+// }
+
+// button[title="Open in CodeSandbox"] {
+// display: none !important;
+// }
+
+// There are a TON of !important to overwrite out-of-the-box custom CSS variables
+
+// component editor
+/* stylelint-disable-next-line selector-class-pattern */
+// .sp-c-ctabmA > .sp-c-kLppIp {
+//   height: 30rem !important;
+// }
```

Comment:
Can we delete this zombie code? 🧟 
Here and the commented code below, since the theme you chose takes care of those issues.

Reply:
It's gone! https://github.com/sparkbox/accessible-components/pull/105/commits/1fa508dd12b7bc45b33ba214d1f04fd1ea5dae89